src/crm.ts

```diff
@@ -0,0 +1,111 @@
+import { ActionMessageV2, EventMessageV2 } from "@proca/queue";
+export type handleResult = {
+  processed: boolean; // if the message has been processed or skipped
+  // there will likely be more attributes to let the CRM inform the queue processor to pause for instance
+};
+
+export { ActionMessageV2 as ActionMessage };
+export { EventMessageV2 as EventMessage };
+
+export type ProcaCampaign = { [key: string]: any }; // TODO export from proca/queue
+
+export type Contact = {
```

Comment:
This does not look right. These types are defined in @proca/queue and will be updated there - prob. they need to be imported not defined here

Reply:
https://github.com/fixthestatusquo/proca-server/blob/main/sdk/queue/src/queue.ts

As I mentioned in the comment, it it indeed something that should be exported from proca/queue, but that's not the case I think.

Could you export and I'll import instead of duplicating?

Reply:
Helpful utility types (operators to modify types and so on): https://www.typescriptlang.org/docs/handbook/utility-types.html

Reply:
````
type Contact = ActionMessageV2['contact']
````